### DISCUSSION TITLE
Is is possible to customize the path to the template?

### DISCUSSION THREAD

**Discussion by nyngwang** [stevearc/overseer.nvim#138] [Q&A] [answered]
Hi, I'm very new to both testing and this beautiful plugin. I only read the nice tutorials.md at the time I'm typing. Based on my current understanding of [this section tutorials.md#build-a-c-file](https://github.com/stevearc/overseer.nvim/blob/master/doc/tutorials.md#build-a-c-file), one can only customize the path with a **fixed** parent `stdpath('config') .. '/lua/overseer/template/'`. Is it possible to customize it?

**Comment by stevearc** [comment, marked as answer]
If you want to load templates by passing in a string, then they must be at the `lua/overseer/template/` path. If you want to load them from somewhere else, you'll just have to `require` it yourself and pass in the definition directly (see "directly registering" of [custom tasks](https://github.com/stevearc/overseer.nvim/blob/master/doc/guides.md#custom-tasks)).

```lua
overseer.register_template(require("path.to.my.template"))
```
